accesscontroltool-oakindex-package/src/main/META-INF/vault/filter.xml

```diff
@@ -14,5 +14,6 @@
 
 <workspaceFilter version="1.0">
     <filter root="/oak:index/repACL" type="cleanup"/><!-- legacy index node name, used prior version 2.7.2  -->
-    <filter root="/oak:index/repACL-custom-1"/>
+    <filter root="/oak:index/repACL-custom-1" type="cleanup"/>
+    <filter root="/oak:index/actool.repACL-custom-1"/>
```

Comment:
This name does not comply with `<prefix>.<customIndexName>-<version>-custom-<customVersion>` either.